lib/compat/wordpress-6.1/block-editor-settings.php

```diff
@@ -13,6 +13,23 @@
  * @return array New block editor settings.
  */
 function gutenberg_get_block_editor_settings( $settings ) {
+	global $post_id;
+	$template_slug    = get_page_template_slug( $post_id );
+	$current_template = gutenberg_get_block_templates( array( 'slug__in' => array( $template_slug ) ) );
```

Comment:
Is it worth us checking whether or not it's a block theme before attempting to grab block templates? (e.g. call `wp_is_block_theme()`?)